projects/dotcms-js/src/lib/core/util/http-response-util.ts

```diff
@@ -12,8 +13,8 @@ export class CwError {
     constructor(
         public code: number,
         public message: string,
-        public request?: Request,
-        public response?: Response,
+        public request?: HttpRequest<any>,
```

Comment:
Let's try to do something with the `any` I know angular use generics for this in the http client, maybe we can do the same thing now.

Reply:
?